src/oxli/hashtable.cc

```diff
@@ -528,6 +632,40 @@ template void Hashtable::consume_seqfile_banding<FastxReader>(
     unsigned long long &n_consumed
 );
 
+//template void Hashtable::consume_seqfile_with_mask<FastxReader>(
+//    std::string const &filename,
+//    Hashtable& mask,
+//    unsigned int &total_reads,
+//    unsigned long long &n_consumed
+//);
+
+
+template void Hashtable::consume_seqfile_with_mask<FastxReader>(
+    ReadParserPtr<FastxReader>& parser,
+    Hashtable* mask,
+    unsigned int &total_reads,
+    unsigned long long &n_consumed
+);
+
+//template void Hashtable::consume_seqfile_banding_with_mask<FastxReader>(
+//    std::string const &filename,
+//    unsigned int num_bands,
+//    unsigned int bands,
```

Comment:
Oops. Forgot the `Hashtable::` namespace prefix in the function definition. Fixed now.